packages/field/demo/stories/FieldStory.tsx

```diff
@@ -16,42 +16,50 @@ import {
 
 interface IComponentProps extends IUseFieldPropGetters {
   isDescribed: boolean;
+  containsMessage: boolean;
```

Comment:
Garden does prefer `is`/`has` naming, and I think `hasMessage` would be more semantically correct as an `<input />` cannot contain a message `<div />`

Reply:
That works, I've update in [7035f95](https://github.com/zendeskgarden/react-containers/pull/473/commits/7035f955591dd0ac74b0bb54528cc68d5ca94400)